.github/pull_request_template.md

```diff
@@ -1,42 +1,21 @@
-<!-- Thank you for submitting a pull request! Find more information in our development guide at https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md and contact us on #dev in Slack. -->
-
+<!--Thank you for submitting a pull request! Find more information in our development guide at https://github.com/trinodb/trino/blob/master/.github/DEVELOPMENT.md and contact us on #dev in Slack. -->
+<!--Provide an overview of the PR for maintainers and reviewers.-->
 ## Description
 
-<!-- Elaborate beyond the title of the PR as necessary to help the reviewers and maintainers.-->
-
-<!-- Answer the following questions to help reviewers and maintainers
-understand this PR's scope at a glance:
--->
-
-> Is this change a fix, improvement, new feature, refactoring, or other?
-
-> Is this a change to the core query engine, a connector, client library, or the SPI interfaces? (be specific)
-
-> How would you describe this change to a non-technical end user or system administrator?
-
-## Related issues, pull requests, and links
 
-<!-- List any issues fixed by this PR, and provide links to other related PRs, upstream release notes, and other useful resources. For example:
-* Fixes #issuenumber
-* Related documentation in #issuenumber
-* [Some release notes](http://usefulinfo.example.com)
--->
 
-<!-- The following sections are filled in by the maintainer with input from the contributor:
-Use :white_check_mark: or (x) to signal selection.
--->
+<!--Provide a user-friendly explanation, keep it brief if it isn't user-visible.-->
+## Non-technical explanation
```

Comment:
Can you share an example of what you think you would like to see here? I can't see the utility of this over the description above.

Reply:
I think for a lot of things, this is going to simply be "Refactor" or "Update tests" or "Upgrade dependency version" or something along those lines, or otherwise a restatement of the description, which is easily understood from a non-technical perspective.

As an arbitrary example where it _would_ be helpful, on #13736, it would be useful to say something like "make sure Trino stops running if it can't secure sufficient resources" in this section, because a less-technical user won't understand what jvmkill is or why terminating the jvm matters.

The section admittedly won't provide a ton of value most of the time, but when it does, it'll help a lot with making sure release notes make sense in situations where the PR author wouldn't otherwise provide a ton of detail as to why users should care about a change.